sno/conflicts.py

```diff
@@ -242,7 +251,8 @@ def conflicts_json_as_geojson(json_obj):
     hidden=True,
     help="Output all conflicts in a flat list, instead of in a hierarchy.",
 )
-def conflicts(ctx, output_format, exit_code, json_style, summarise, flat):
+@click.argument("args", nargs=-1)
```

Comment:
Check `sno conflicts --help` for this, probably improve it a bit.

 * call it `filters` rather than args
 * maybe set `metavar` to something, if it makes the help text more readable: https://click.palletsprojects.com/en/7.x/documentation/#meta-variables
 * Add `help` with a filter example

Reply:
Done, also modified commit.py and diff.py which are the other commands that currently support filters.